teraserver/python/services/BureauActif/API/QueryLoginType.py

```diff
@@ -29,11 +39,38 @@ def get(self):
         if current_login_type == LoginType.PARTICIPANT_LOGIN:
             login_infos['login_type'] = 'participant'
             login_infos['login_id'] = current_participant_client.id_participant
+            params = {'participant_id': current_participant_client.id_participant}
+            endpoint = '/api/service/participants'
+            response = Globals.service.get_from_opentera(endpoint, params)
+
+            if response.status_code == 200:
+                participant_infos = response.json()
+                login_infos['username'] = participant_infos['participant_username']
 
         if current_login_type == LoginType.USER_LOGIN:
             login_infos['login_type'] = 'user'
             login_infos['login_id'] = current_user_client.id_user
             login_infos['is_super_admin'] = current_user_client.user_superadmin
+            params = {'user_id': current_user_client.id_user}
+            endpoint = '/api/service/users'
+            response = Globals.service.get_from_opentera(endpoint, params)
 
-        return login_infos
+            if response.status_code == 200:
+                user_infos = response.json()
+                login_infos['username'] = user_infos['user_username']
+
+        # If reservation has a session associated to it, get it from OpenTera
+        if args['id_project'] or args['id_site']:
+            if args['id_project']:
+                params = {'id_project': args['id_project'], 'uuid_user': current_user_client.user_uuid}
+            if args['id_site']:
+                params = {'id_site': args['id_site'], 'uuid_user': current_user_client.user_uuid}
 
+            endpoint = '/api/service/user_role'
+            response = Globals.service.get_from_opentera(endpoint, params)
+
+            if response.status_code == 200:
+                permission_info = response.json()
+                login_infos.update(permission_info)
```

Comment:
Here, the fields "project_admin" and "site_admin" could be created with "project_role" and "site_role", to prevent changes in the "BureauActif" frontend. Since this is an API local to the service, this would be OK to do so here.